easybuild/scripts/rpath_args.py

```diff
@@ -93,6 +93,8 @@
 
     # filter out --enable-new-dtags if it's used;
```

Comment:
Ah, ok, good point...

Maybe clarify that with a comment here, since I suspect otherwise people (e.g. me) will go wonder again why we don't just drop `--enable-new-dtags` or why we possible add `--disable-new-dtags` multiple times... :)